src/main/java/com/facebook/openwifirrm/mysql/DatabaseManager.java

```diff
@@ -482,6 +482,8 @@ public void addWifiScan(
 				stmt.setLong(4, entry.last_seen);
 				stmt.setInt(5, entry.signal);
 				stmt.setInt(6, entry.channel);
+				// TODO: untested
+				stmt.setLong(7, entry.unixTimeMs);
```

Comment:
How do I test this and the other untested TODO in this file?

Reply:
Just to clarify, what do you mean by "omit this"? I'm guessing you mean to keep the `setLong()` line but not include the `TODO` (since it will be consumed as part of another existing `TODO`), is that right?

Reply:
No, I mean to remove any changes to this file and deal with database cleanup later (this PR is already large enough).